src/Microsoft.TestPlatform.ObjectModel/Navigation/DiaSession.cs

```diff
@@ -95,6 +95,10 @@ public INavigationData GetNavigationDataForMethod(string declaringTypeName, stri
         private static ISymbolReader GetSymbolReader(string binaryPath)
         {
             var pdbFilePath = Path.ChangeExtension(binaryPath, ".pdb");
+
+            // For remote scenario, also look for pdb in current directory, (esp for UWP)
+            // The alternate search path should be an input from Adapters, but since it is not so currently adding a HACK
```

Comment:
Look in `searchPath` too if not found in current directory?

Reply:
I agree, we will bring this in later